index.html

```diff
@@ -208,142 +193,207 @@ <h2>
           </dt>
           <dd>
             <p>
-              When invoked, returns a Promise object representing a request for
-              access to MIDI devices on the user's system.
+              When invoked the {{Navigator/requestMIDIAccess()}} method returns
+              a promise that represents a request for access to MIDI devices on
+              the user's system.
             </p>
             <p>
               Requesting MIDI access SHOULD prompt the user for access to MIDI
-              devices, particularly if system exclusive access is requested. In
-              some scenarios, this permission may have already been implicitly
-              or explicitly granted, in which case this prompt may not appear.
-              If the user gives express permission or the call is otherwise
-              approved, the vended Promise is resolved. The underlying system
-              may choose to allow the user to select specific MIDI interfaces
-              to expose to this API (i.e. pick and choose interfaces on an
-              individual basis), although this is not required. The system may
-              also choose to prompt (or not) based on whether system exclusive
-              support is requested, as system exclusive access has greater
-              privacy and security implications.
+              devices, particularly if [=system exclusive=] access is requested.
+              In some scenarios, this permission may have already been
+              implicitly or explicitly granted, in which case this prompt may
+              not appear. If the user gives [=express permission=] or the call
+              is otherwise approved, the vended Promise is resolved. The
```

Comment:
Similarly, this is already in the algorithm. 

```suggestion
              not appear. The
```